src/class/IContentTypes/IContentTypes.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef I_CONTENT_TYPES_HPP
+#define I_CONTENT_TYPES_HPP
+
+#include <iostream>
+#include <list>
+#include <map>
+#include <string>
+
+#ifdef TEST_ON
+#define private public
+#define protected public
+#endif
+
+class IContentTypes
+{
+  public:
+	static std::map<std::string, std::string> _content_types;
```

Comment:
make it const since this will never change in runtime
static std::map<std::string, std::string> -> static const std::map<std::string, std::string>

_content_types -> content_types
("_" is for private)

Reply:
I change it for the next PR